x-pack/platform/test/api_integration/apis/upgrade_assistant/index.ts

```diff
@@ -5,6 +5,7 @@
  * 2.0.
  */
 
+// @ts-ignore TS error ignored for package.json import
 import { version as kibanaVersion } from '../../../../../package.json';
```

Comment:
I gave up to find a solution to make TS Check passing. Open for any suggestions.